client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -225,6 +245,15 @@ const AccountOverviewPage = () => {
 										subscription={subscription}
 									/>
 								))}
+							{featureSwitches.singleContributions &&
+								category === 'recurringSupport' &&
```

Comment:
Minor the heading above the card on the design has 'Your Support' instead of 'Recurring Support'

Reply:
Will ask Daniel as well about this

Reply:
We're probably going to rename the "Recurring Support" header to "Your Support", but will do this in a subsequent PR on confirmation.